.changeset/sad-dodos-sell.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/fluid-static": minor
+---
+---
+"section": deprecation
+---
+
+Marked `IContainerRuntimeOptions.enableGroupedBatching` as deprecated
+
+- We will remove the ability to disable Grouped Batching in v2.20.0. The only exception (i.e. where Grouped Batching would be disabled) is for compatibility with older (v1) clients, and this will be implemented without needing to expose `IContainerRuntimeOptions.enableGroupedBatching`.
```

Comment:
```suggestion
The `IContainerRuntimeOptions.enableGroupedBatching` property is deprecated and will be removed in version 2.20.0. This will mean that the grouped batching feature can no longer be disabled. In versions 2.20.0 and beyond, grouped batching is required for the proper functioning of the Fluid Framework.

The sole case where grouped batching will be disabled is for compatibility with older v1 clients, and this will be implemented without any need for the configurable `IContainerRuntimeOptions.enableGroupedBatching` option.
```

Reply:
Done